notebooks/cva_opt_with_channels_example.ipynb

```diff
@@ -0,0 +1,435 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "7463a182-7838-4909-892a-0be486bd8800",
+   "metadata": {},
+   "source": [
+    "![](../images/TQ42_Banner.png)\n",
+    "\n",
+    "\n",
+    "# Welcome to TQ42!\n",
+    "\n",
+    "## Example how to use channels with CVA_OPT\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "dcd8d3ef-3069-4489-8397-8cba3b3ee769",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from tq42.client import TQ42Client\n",
+    "from tq42.experiment_run import ExperimentRun\n",
+    "from tq42.organization import list_all as list_all_organizations\n",
+    "from tq42.project import list_all as list_all_projects\n",
+    "from tq42.experiment import list_all as list_all_experiments\n",
+    "from tq42.experiment_run import ExperimentRunStatusProto\n",
+    "from tq42.algorithm import AlgorithmProto\n",
+    "from tq42.compute import HardwareProto\n",
+    "import json\n",
+    "import pandas\n",
+    "import asyncio\n",
+    "import numpy as np\n",
+    "from tq42.channel import Channel, Ask, Tell\n",
+    "import OptimizationTestFunctions as otf"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c9ed1ab2-279f-4079-8719-c2f2fd18847b",
+   "metadata": {},
+   "source": [
+    "### Get your experiment id by descending the organization, project, experiment hierarchy"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "8c4d62de633fd4c1",
+   "metadata": {
+    "collapsed": false,
+    "jupyter": {
+     "outputs_hidden": false
+    }
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "{\"device_code\":\"v-YmkmEoO3Hny9AkO9DZ-eTg\",\"user_code\":\"CRWR-ZNTN\",\"verification_uri\":\"https://auth.terraquantum.io/activate\",\"expires_in\":900,\"interval\":5,\"verification_uri_complete\":\"https://auth.terraquantum.io/activate?user_code=CRWR-ZNTN\"}\n",
```

Comment:
you want to clean the outputs (or at least these corresponding to authentication data)

Reply:
Yes, I will clear all outputs.